adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/OAuth2/TokenResponse.cs

```diff
@@ -35,6 +35,7 @@
 using System.Text;
 using Microsoft.Identity.Core;
 using Microsoft.Identity.Core.Cache;
+using Microsoft.Identity.Core.Helpers;

```

Comment:
>Helpers [](start = 30, length = 7)

is this needed? should the helper below be removed and other references updated in this file?